webpack.config.js

```diff
@@ -0,0 +1,96 @@
+const path = require('path');
```

Comment:
I am wondering why did you ditch react-scripts for your custom Webpack configuration?